app/layouts/error.vue

```diff
@@ -19,7 +19,20 @@
 <script>
 export default {
   layout: 'error',
-  props: ['error'] // you can set a custom layout for the error page
+  props: ['error'],
+  data () {
+    return {
+      tryReloadInterval: null
+    }
+  },
+  mounted () {
+    this.tryReloadInterval = setInterval(() => {
```

Comment:
So now on error page we try the original request again after 5 seconds? That's awesome addition if so!!